### PR TITLE
[web]: update libraries.json to use the include feature.

### PR DESCRIPTION
**Pull request by sigmundch**
Recently the library specification format was changed to allow
refering to other specification files. This will make changes to
the underlying dart2js/ddc platform configuration to roll out automatically
to flutter in the future.

/cc FYI - @johnniwinther @rileyporter

### CODE REVIEW

**Review by johnniwinther — APPROVED** [review]

**Comment by sigmundch** [comment]
@yjbanov - I'll wait for your approval before merging.

**Review by yjbanov — APPROVED** [review]
lgtm

**Pull request merged**
